pbspro.spec

```diff
@@ -194,7 +196,9 @@ HPC clusters, clouds and supercomputers.
 This package is intended for a client host and provides
 the PBS Professional user commands.
 
-%if 0%{!?opensuse_bs}
+%if 0%{?opensuse_bs}
```

Comment:
The problem was identified while building and testing this ticket, but it's not strictly related. Seemed like a lot of overhead to file a separate ticket, so I just made the change here.